src/app/api/client/route.ts

```diff
@@ -0,0 +1,20 @@
+import { NextResponse, NextRequest } from 'next/server'
+
+export async function GET(req: NextRequest) {
+  const clientId = req.nextUrl.searchParams.get('clientId') as string
+
+  const options = {
+    method: 'GET',
+    headers: {
+      accept: 'application/json',
+      'X-API-KEY': process.env.COPILOT_API_KEY as string,
+    },
+  }
+
+  const res = await fetch(
+    `https://api-beta.copilot.com/v1/clients/${clientId}`,
```

Comment:
why is this api url static? is this same for all envs? would it be safe to have an env variable ?